docs/api/crash-reporter.md

```diff
@@ -45,7 +45,7 @@ The `crashReporter` module has the following methods:
   * `ignoreSystemCrashHandler` Boolean (optional) - Default is `false`.
   * `extra` Object (optional) - An object you can define that will be sent along with the
     report. Only string properties are sent correctly. Nested objects are not
-    supported.
+    supported. The property name and the value must be less than 63 characters.
```

Comment:
I think they must be less than 64 characters, looks like the code does a `if (name.length() > kNameMaxLength - 1)` which means it will only be truncated if the name/value is greater than 63, not equal to 63.